install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
+configured automatically. However, iptables rules can break {product-title} if
+not configured correctly. The advantages of firewalld include allowing multiple
+objects to safely share the firewall rules. However, it is not available on RHEL
+Atomic Host.
+
+To use firewalld as the firewall for an {product-title} install, add the
+`os_firewall_use_firewalld` variable to the list of configuration variables in
+Ansible host file on install:
+
+----
+[OSEv3:vars]
+os_firewall_use_firewalld=True
+----
+
```

Comment:
We definitely want to move to a world where everyone is using firewalld; OpenShift+iptables-service is always going to be flaky (and is less shiny-nftables-future-proof too).

From our POV, having most people using firewalld but some still using iptables-service for whatever reason is better than having everyone using iptables-service (because it means fewer people shooting themselves in the foot). But maybe for other teams (installer, docs, etc) it's better to have everyone using the same thing...